src/components/Home/NewBookForm/NewBookForm.css

```diff
@@ -1,21 +1,27 @@
 .new-book-form {
   display: flex;
   flex-direction: column;
-  width: 50%;
+  width: 100%;
   min-width: 400px;
 }
 
```

Comment:
- [ ] Kindly make sure to align your form inputs with the `flex-direction: row` in order to match the provided Zeplin design. It could be better to align the form inputs with the `flex-direction: column` in the mobile version. Kindly fix that.


    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/230624828-abb80697-6a33-4222-a41c-1ca730520a64.png)

    - ### ZEPLIN DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/230624894-0b36663d-9d7a-42f6-9618-eef8a7145cae.png)